Header header
int16 exercise_id
float32 t_from_s
float32 t_to_s
int16 reps
bool start
